src/main/java/com/org/trustservice/model/TrustGroupPermission.java

```diff
@@ -0,0 +1,60 @@
+package com.org.trustservice.model;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import org.hibernate.annotations.GenericGenerator;
+
+import com.org.trustservice.model.id.TrustGroupPermissionId;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Table(name = "trustgroup_permissions_def")
+@Entity
+@IdClass(TrustGroupPermissionId.class)
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class TrustGroupPermission {
+
+    @Id
+    @GeneratedValue(generator = "uuid")
+    @GenericGenerator(name = "uuid", strategy = "org.hibernate.id.UUIDGenerator")
+    private UUID id;
```

Comment:
inconsistency in naming.
Either use id all across or {className}Id all across.
i.e. id or trustGroupPermissionId
but it has to be consistent